micropip/_cached_version.py

```diff
@@ -0,0 +1,44 @@
+from ._vendored.packaging.src.packaging.version import InvalidVersion, Version
+
+__all__ = ["CachedVersion", "parse_cached_version", "InvalidVersion"]
```

Comment:
We won't do `from _cached_version import *` so I think this `__all__` attribute in unnecessary.